Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -323,4 +325,11 @@ - (void)locationManager:(CLLocationManager *)manager didFailWithError:(NSError *
   _locationManager.desiredAccuracy = RCT_DEFAULT_LOCATION_ACCURACY;
 }
 
+- (void)assertLocationEnabled
```

Comment:
just a small nit: this method is not doing what name suggests. Maybe `checkLocationConfig`?